### ISSUE TITLE
Show-queue refresh not adding existing files

### ISSUE THREAD

**Issue by kizza2153** [SickChill/SickChill#6097]
Also having the same issue with TVDB, but does NOT seem to be my issue. Have been having this problem for over a week.

**Branch/Commit:** 
Branch: master
Commit: b2d7ac109604ed7585ba0b56c53fcf93e4f2fadd
Database Version: 44.4

**OS:** 
Unraid 6.8.2 running in 'linuxserver' docker

**Browser:** 
Chrome/Firefox

**What you did:** 
Added existing TV files to Sickchill after migrating server from Windows 10 to Unraid. Sickchill is running in a 'linuxserver' docker and doesn't seem to have any issues apart from this.

**What happened:**
Around 75% of my files added successfully. There doesn't seem to be any relationship between the files that were added and ones that weren't. Have tried to change ownership of files and permissions, but didn't make a difference. Logs seem to indicate that they were added successfully but they still remain as not downloaded in Sickchill.

**What you expected:**

For all my existing files to be added to Sickchill.

Logs:
```
2020-03-15 09:07:06 DEBUG    WEBSERVER-HOME :: Queueing show refresh for Chernobyl
2020-03-15 09:07:08 INFO     SHOWQUEUE-REFRESH :: Performing refresh on Chernobyl
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Loading all episodes from the show directory /tv/Chernobyl
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode from Chernobyl - S01E01 - 12345.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode object from /tv/Chernobyl/Season 01/Chernobyl - S01E01 - 12345.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: /tv/Chernobyl/Season 01/Chernobyl - S01E01 - 12345.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: /tv/Chernobyl/Season 01/Chernobyl - S01E01 - 12345.mkv parsed to Chernobyl S01E01
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Setter sets location to /tv/Chernobyl/Season 01/Chernobyl - S01E01 - 12345.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: Chernobyl - S01E01 - 12345
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode from Chernobyl - S01E02 - Please Remain Calm.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode object from /tv/Chernobyl/Season 01/Chernobyl - S01E02 - Please Remain Calm.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: /tv/Chernobyl/Season 01/Chernobyl - S01E02 - Please Remain Calm.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: /tv/Chernobyl/Season 01/Chernobyl - S01E02 - Please Remain Calm.mkv parsed to Chernobyl S01E02
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Setter sets location to /tv/Chernobyl/Season 01/Chernobyl - S01E02 - Please Remain Calm.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: Chernobyl - S01E02 - Please Remain Calm
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode from Chernobyl - S01E03 - Open Wide, O Earth.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode object from /tv/Chernobyl/Season 01/Chernobyl - S01E03 - Open Wide, O Earth.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: /tv/Chernobyl/Season 01/Chernobyl - S01E03 - Open Wide, O Earth.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: /tv/Chernobyl/Season 01/Chernobyl - S01E03 - Open Wide, O Earth.mkv parsed to Chernobyl S01E03
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Setter sets location to /tv/Chernobyl/Season 01/Chernobyl - S01E03 - Open Wide, O Earth.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: Chernobyl - S01E03 - Open Wide, O Earth
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode from Chernobyl - S01E04 - The Happiness of All Mankind.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode object from /tv/Chernobyl/Season 01/Chernobyl - S01E04 - The Happiness of All Mankind.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: /tv/Chernobyl/Season 01/Chernobyl - S01E04 - The Happiness of All Mankind.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: /tv/Chernobyl/Season 01/Chernobyl - S01E04 - The Happiness of All Mankind.mkv parsed to Chernobyl S01E04
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Setter sets location to /tv/Chernobyl/Season 01/Chernobyl - S01E04 - The Happiness of All Mankind.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: Chernobyl - S01E04 - The Happiness of All Mankind
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode from Chernobyl - S01E05 - Vichnaya Pamyat.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Creating episode object from /tv/Chernobyl/Season 01/Chernobyl - S01E05 - Vichnaya Pamyat.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: /tv/Chernobyl/Season 01/Chernobyl - S01E05 - Vichnaya Pamyat.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: /tv/Chernobyl/Season 01/Chernobyl - S01E05 - Vichnaya Pamyat.mkv parsed to Chernobyl S01E05
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Setter sets location to /tv/Chernobyl/Season 01/Chernobyl - S01E05 - Vichnaya Pamyat.mkv
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Using cached parse result for: Chernobyl - S01E05 - Vichnaya Pamyat
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: 360893: Loading all episodes with a location from the database
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Checking if /tv/Chernobyl/poster.jpg exists: True
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Checking if /tv/Chernobyl/banner.jpg exists: True
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Checking if /tv/Chernobyl/season01-poster.jpg exists: True
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Metadata provider KODI 12+ creating season banners for Chernobyl
2020-03-15 09:07:08 DEBUG    SHOWQUEUE-REFRESH :: Checking if /tv/Chernobyl/season01-banner.jpg exists: False
```

**Comment by marcb1387** [comment]
yeah tvdb is having all kinda issues, im also having trouble with plex for something and others with kodi too. https://forums.thetvdb.com/viewtopic.php?f=122&t=63148

**Comment by kizza2153** [comment]
I don't think it's TVDB related. It had already got the details in the past and they are all present. It seems to just not link the existing files to episodes even though it does find them

**Comment by miigotu** [comment]
It's very likely to be tvdb related, but have you tried a restart after adding them? How are you adding them? The show should not exist in SC, then you click add show -> add existing.

**Comment by kizza2153** [comment]
I've tried a restart as well as removing them and re-adding. Originally, I added my whole TV folder in to a new, blank install of sickchill. It went through and added all the folders and created every TV show. However, it was very hit and miss with individual episodes. With some TV shows it found every episode I had in each folder. With other shows, it found some episodes but not others. Others again (like Chernobyl which I posted the logs for), it found the existing folder and created the show on sickchill, but didn't recognise any of the existing video files so it is saying I have downloaded 0/5 episodes.

There is no difference in folder structure between shows and they all appeared in Sickchill when it was running on Windows 10.
